app/models/api/fake_data/grantee.rb

```diff
@@ -0,0 +1,20 @@
+require "ffaker_wrapper"
+
+class Api::FakeData::Grantee
```

Comment:
[suggestion] we probably want to do something similar to the ActivityReport where we pass in the ID to an initializer so it can be consistent with the grantee params